shared/README.md

```diff
@@ -54,11 +55,11 @@ flowchart TD
         subgraph Pocket's Application Specific Bus
             B("Bus")
             E("Main Events Channel")
-            Clock("Clock")
             B <-.-> E
-            B <-.-> Clock
         end
         subgraph Pocket's Core Modules
+            RMGR(RuntimeMgr)
```

Comment:
I don't think it's relevant at this level, sounds like an implementation detail. 
It would make the diagram quite "busy" IMHO, also I could argue that `config` and `genesis` are probably more important than clock in the context of `runtimeMgr`